corpus_merge.sh

```diff
@@ -103,7 +103,7 @@ mkdir "${CACHE_DIR}/${FUZZ_TARGET}"
 coverage=0
 if [[ -n $(ls "${DEST_DIR}") ]]; then
 	cp "${DEST_DIR}"/* "${CACHE_DIR}/${FUZZ_TARGET}/"
-	coverage=$(measure_coverage)
+	coverage=$(measure_coverage | sed 's/[^0-9]*//g')
```

Comment:
What if the current directory has numbers in it?

Also, can we move the stripping of CWD into `measure_coverage` itself?